schema/types/isolator.go

```diff
@@ -69,8 +112,17 @@ func (is *Isolators) Unrecognized() Isolators {
 // serialized as any arbitrary JSON blob. Specific Isolator types should
 // implement this interface to facilitate unmarshalling and validation.
 type IsolatorValue interface {
+	// UnmarshalJSON unserialize a JSON-encoded isolator
 	UnmarshalJSON(b []byte) error
+	// AssertValid returns a non-nil error value if an IsolatorValue is not valid
+	// according to appc spec
 	AssertValid() error
+	// MaxInstances specifies the maximum number of allowed isolator instances
+	// for this isolator type
+	MaxInstances() uint
```

Comment:
I don't feel particularly strongly about this, but generally speaking in Go we err on the side of minimal possible interfaces (including for function prototypes) until something more complex is demonstrably necessary - since typically dependencies are vendored and builds are static, so refactorings are straightforward. 

Similar logic applies when considering whether to export methods (e.g. in this case `maxInstances`/`multipleAllowed` can probably remain package-local for now)

https://github.com/appc/spec/pull/621/files#r67835328 is similar